test/manual/mongodb_oidc_azure.prose.test.ts

```diff
@@ -0,0 +1,202 @@
+import { expect } from 'chai';
+
+import {
+  Collection,
+  CommandFailedEvent,
+  CommandStartedEvent,
+  CommandSucceededEvent,
+  MongoClient,
+  OIDC_WORKFLOWS
+} from '../mongodb';
+
+describe('OIDC Auth Spec Prose Tests', function () {
+  const callbackCache = OIDC_WORKFLOWS.get('callback').cache;
+  const azureCache = OIDC_WORKFLOWS.get('azure').cache;
+
+  describe('3. Azure Automatic Auth', function () {
+    let client: MongoClient;
+    let collection: Collection;
+
+    afterEach(async function () {
+      await client?.close();
+    });
```

Comment:
We connect in beforeAll but close in afterEach, I think we're passing b/c there's only one test per describe block, I guess that won't change 🤔 but seems like something we should just balance (each in both)

Reply:
Everything now uses `beforeEach`/`afterEach` only.